modules/claims_api/lib/claims_api/evss_status_mapper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Maps a given 'status' to a known enum status
+
+module ClaimsApi
+  class EvssStatusMapper
```

Comment:
The upstream service we're calling to get these values is actually BGS (not EVSS).  So we should prob rename this class to something like `BGSClaimStatusMapper`